.travis.yml

```diff
@@ -1,6 +1,3 @@
 language: node_js
 node_js:
   - "7"
-  - "6"
```

Comment:
My opinion is: it depends.

Are we transpiling the code anyways? When someone downloads our npm package it uses the transpiled code?

If both of those questions are correct then yes.